ch3/recursion.js

```diff
@@ -0,0 +1,32 @@
+const assert =  require('assert');
```

Comment:
Because my linter in vscode makes colors for suggestions and I want them to make go away. This one says you can change the file to an ES module and change the syntax so I just accept the quick fix.

Reply:
i changed it back and read your SO discussion. Thank you